django_unused_media/utils.py

```diff
@@ -23,3 +27,13 @@ def get_file_fields():
                 fields.append(field)
 
     return fields
+
+
+def file_age_sec(path_to_file):
+    if platform.system() == 'Windows':
+        mtime = os.path.getmtime(path_to_file)
+    else:
+        stat = os.stat(path_to_file)
+        mtime = stat.st_mtime
+
+    return time.time() - mtime
```

Comment:
Maybe this should be relative to the tool's start, not "now" at the time it's queried.

While I don't think this is a significant issue because files are scanned before the database, the behavior may be surprising if the scan is slow and finds more files than expected because the age cutoff moves in realtime.